src/AccessibilityInsights.Extensions.GitHub/IssueReporter.cs

```diff
@@ -0,0 +1,70 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+using AccessibilityInsights.Extensions.AzureDevOps;
+using AccessibilityInsights.Extensions.Interfaces.IssueReporting;
+using Newtonsoft.Json;
+using System;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+
+namespace AccessibilityInsights.Extensions.GitHub
+{
+    public class IssueReporter : IIssueReporting
+    {
+        private IssueReporter _instance;
+        private ConfigurationModel _configurationControl;
+
+        private IssueReporter()
+        {
+            _configurationControl = new ConfigurationModel();
+        }
+
+        public IssueReporter getDeafualtInstance()
```

Comment:
Typo. Should be "Default"

Reply:
get => Get

---
In reply to: [267042274](https://github.com/Microsoft/accessibility-insights-windows/pull/184#discussion_r267042274) [](ancestors = 267042274)